src/settings.ts

```diff
@@ -205,6 +207,18 @@ export async function parseSettings(
     }
   });
 
+  const dumpCommand = await check(
+    "dumpCommand",
+    async (dumpCommand: string) => {
+      if (!dumpCommand || typeof dumpCommand == "string") {
+        return dumpCommand;
+      } else {
+        throw new Error(
+          `Expected dumpCommand to be a string if present; recieved '${typeof dumpCommand}'`
+        );
+      }
+    }
+  );
```

Comment:
If we go the hooks route, then this will be `afterAllMigrations` and it should be factored into the `afterReset` code above.